server/api/order.js

```diff
@@ -0,0 +1,23 @@
+const router = require('express').Router()
+const Order = require('../db/models/Order')
+const {requireToken, isAdmin} = require('./gatekeepingMiddleware')
+
+router.get('/', requireToken, isAdmin, async (req, res, next) => {
+  try {
+    const orders = await Order.findAll()
+    res.send(orders)
+  } catch (err) {
+    next(err)
+  }
+})
+
+router.get('/:orderid', async (req, res, next) => {
```

Comment:
This route should have token checking, and only send the order back if the order's userId matches the token's ID (and a 403 otherwise). I also don't see your orders routes being used in your index file, but I could be missing something